### PR TITLE
2-week-out blog post

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

**Comment by plaindocs** [comment]
I fixed that wot I saw wot needed fixing. The rest LGTM.

File: docs/conf/eu/2017/news/2week-info.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+========================================
+
+We hope everyone is sufficiently excited for the conference, which is now just about two weeks away!
+
+Schedule and Activities Summary
+-------------------------------
+
+Our website is up-to-date with the latest details about sessions, activities, and locations. However, we are also happy to share with you a quick summary right here:
+
+- **Saturday September 9th - Boat Tour of Prague**. We will meet at `Prague Boats, pier no. 5 <https://goo.gl/maps/bqLP3VaytVo>`_ on **Saturday, September 9 at 13:45** to board the boat. We must leave the pier at **14:00** sharp (captain's rules) so unfortunately we cannot wait for late arrivals. All the details are in our `Boat Tour <http://www.writethedocs.org/conf/eu/2017/boat/>`_ page.
```
Comment (ericholscher) [inline docs/conf/eu/2017/news/2week-info.rst]: Probably worth mentioning it's a seperate ticket.

File: docs/conf/eu/2017/news/2week-info.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+For practical information about Prague, check out the `Visiting Prague <http://www.writethedocs.org/conf/eu/2017/visiting/>`_
+
+And if you still have questions, perhaps you'll find the answers in our `Welcome Wagon <http://www.writethedocs.org/conf/eu/2017/welcome-wagon/>`_ guide. You're also welcome to reach out to @praguewelcomewagon on Slack or send us an email at europe@writethedocs.org.
+
+Badges and Shirts Registration Ends August 31st
+-----------------------------------------------
+
+Even though we still have a few `tickets <http://www.writethedocs.org/conf/eu/2017/tickets>`_ left, we are getting closer to the cut-off time for badges and shirts.
+Please note that if you purchase tickets after **August 31st, 2017**, we will not be able to guarantee a badge or a shirt.
```
Comment (ericholscher) [inline docs/conf/eu/2017/news/2week-info.rst]: We should probably mention a badge with your name printed, they will still have one, just with a hand written name.

**Review by ericholscher — COMMENTED** [review]
A couple nits, otherwise looks good.

File: docs/conf/eu/2017/news/2week-info.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Schedule and Activities Summary
+-------------------------------
+
+Our website is up-to-date with the latest details about sessions, activities, and locations. However, we are also happy to share with you a quick summary right here:
+
+- **Saturday September 9th - Boat Tour of Prague**. We will meet at `Prague Boats, pier no. 5 <https://goo.gl/maps/bqLP3VaytVo>`_ on **Saturday, September 9 at 13:45** to board the boat. We must leave the pier at **14:00** sharp (captain's rules) so unfortunately we cannot wait for late arrivals. All the details are in our `Boat Tour <http://www.writethedocs.org/conf/eu/2017/boat/>`_ page.
+
+- **Sunday September 10th - Writing Day and Welcome Reception**. Both the `Writing Day <http://www.writethedocs.org/conf/eu/2017/writing-day/>`_ and the Welcome Reception will be held at the main conference venue, `Auto Klub, Opletalova 1337/29, Praha 1 <https://goo.gl/maps/dcbYPV5x5DN2>`_. We will also hold a `Welcome Wagon <http://www.writethedocs.org/conf/eu/2017/welcome-wagon/>`_ orientation during the receiption to help ease your way into the conference.
```
Comment (ericholscher) [inline docs/conf/eu/2017/news/2week-info.rst]: Maybe expand a bit more about the writing day? "Come with your laptop and work with others on documenting some open source projects, or your own"?

Comment (thatdocslady) [reply docs/conf/eu/2017/news/2week-info.rst]: We already wrote this in other places, I am trying to keep the post from getting long-winded, and we are already linking to the full writing day description...

**Pull request merged**